BackofficeBundle/Resources/public/smartadmin/js/app.js

```diff
@@ -762,6 +762,11 @@ $.fn.extend({
 				} // end else
 			} // end if
 		});
+
+		// prevent url load on extends accordion menu by click on collapse sign

```

Comment:
I totally agree, the directory public/smartadmin will be override on the next version of smartadmin, and your code will be then lost.